### PR TITLE
fix: add rejected_msg field to limit-req plugin form

### PR DESCRIPTION
**Pull request by oil-oil**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Add rejected_msg field to limit-req plugin form.
image

**Related issues**

https://github.com/apache/apisix-dashboard/issues/2310

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by zaunist** [comment]
It's better to add some test case.

**Comment by guoqqqi** [comment]
Hi, @oil-oil add some pictures of your changes in the description section to make it easier to review 😄

**Comment by oil-oil** [comment]
> It's better to add some test case.

OK,I'll do it.

**Comment by zaunist** [comment]
Could you provide some screenshoot about your change?

**Comment by oil-oil** [comment]
I added a new field to the form.

before:
image

after:
image

**Review by zaunist — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

File: web/src/components/Plugin/UI/limit-req.tsx [hunk 1]
```diff
@@ -102,6 +102,14 @@ const LimitReq: React.FC<Props> = ({ form, schema }) => {
           max={properties.rejected_code.maximum}
         />
       </Form.Item>
+      <Form.Item
+        label="rejected_msg"
+        name="rejected_msg"
+        initialValue={properties.rejected_msg.default}
```
Comment (bzp2010) [inline web/src/components/Plugin/UI/limit-req.tsx:108]: According the schema definition, the `rejected_msg` has no default value.

**Review by bzp2010 — APPROVED** [review]

**Comment by bzp2010** [comment]
If the CI test passes it's ready.

**Comment by oil-oil** [comment]
> If the CI test passes it's ready.

@bzp2010  I can pass the test in the development environment, can you re-execute the CI test for me?

**Review by Baoyuantop — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**